README.md

```diff
@@ -0,0 +1,62 @@
+##  :memo: Description
```

Comment:
It's missing the `.gitignore` file, you need to commit and push it as well, otherwise some files like `node_modules` would be listed as created/changed on my computer because I don't have that file.

Reply:
No worries, I didn't know that. I'll do it.